workflow/main.rb

```diff
@@ -29,7 +38,7 @@ def emoji_item(emoji)
     query = Regexp.escape(ARGV.first)
 
     items = Emoji.all.map(&:name).grep(/#{query}/).each do |code|
-      fb.add_item(emoji_item(Emoji.find_by_alias(code)))
+      fb.add_item(emoji_item(Emoji.find_by_alias(code))) unless is_image_only(Emoji.find_by_alias(code))
```

Comment:
Good catch. I usually do that any time I have to reuse returned data but I missed it here. Doh!